profile/README.md

```diff
@@ -42,20 +18,14 @@ We're always looking for passionate contributors! 🤝 Whether it's for developm
 
 ---
 
-## 🛠️ **Technologies Used**
+## 🏗️ **Overview of the architecture**
+
+![image](https://github.com/user-attachments/assets/c72a3a51-bfc0-4e3d-9cde-6ae79cdb73dd)
```

Comment:
Very sad you didn't manage to make mermaid schema

Reply:
it sucks